test/rmgpy/chemkinTest.py

```diff
@@ -993,13 +993,13 @@ def setUpClass(cls):
         ]
         cls.template_list = [
             ["C_methyl", "C_methyl"],
-            ["C/H3/Cs\H3", "C_methyl"],
-            ["C/H3/Cs\H3", "C_rad/H2/Cs"],
-            ["C/H3/Cs\H3", "H_rad"],
+            [r"C/H3/Cs\H3", "C_methyl"],
```

Comment:
What is the r for? regex? Why only adding for the first one in the list?

Reply:
`r` stands for raw - see [this discussion](https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2504#discussion_r1273868680 with @kspieks above - the `\H` is being interpreted as an escape character so the raw string tells python to not do that